config/prod/iatlas-db.yaml

```diff
@@ -10,5 +10,5 @@ parameters:
   KmsKey: !stack_output_external iatlas-kms-prod::KmsKey
   DbSecretPrefix: 'prod_'
   DbCredsName: 'iatlas_db_creds_encrypted_prod'
-  DBInstanceClass: 'db.t3.large'
+  DBInstanceClass: 'db.r6g.large'
```

Comment:
Why not R7g or T4? https://aws.amazon.com/rds/instance-types/

Reply:
We are needing memory optimized. It looks like the `r7g` is a great option, has a bit more network bandwidth, but is also a little bit more expensive. Thoughts on why the `r7g` over the `r6g`?

Reply:
Yeah, I noticed that the last models are not cheaper than the previous ones nearly across the board. Unless you need the extra perf, I think it's fine.